api/peer/peertest/list.go

```diff
@@ -64,7 +65,7 @@ func (_m *MockChooser) Choose(_param0 context.Context, _param1 *transport.Reques
 
 // Choose indicates an expected call of Choose
 func (_mr *MockChooserMockRecorder) Choose(arg0, arg1 interface{}) *gomock.Call {
-	return _mr.mock.ctrl.RecordCall(_mr.mock, "Choose", arg0, arg1)
+	return _mr.mock.ctrl.RecordCallWithMethodType(_mr.mock, "Choose", reflect.TypeOf((*MockChooser)(nil).Choose), arg0, arg1)
```

Comment:
These files should not be changed by this. It looks like gomock got updated recently and the generated code is changing. Can you do that separately? Just a simple,

```
glide up
make generate
```